cmd/buildah/addcopy.go

```diff
@@ -20,63 +21,56 @@ type addCopyResults struct {
 	contextdir string
 }
 
+func createCommand(addCopy string, desc string, short string, opts *addCopyResults) *cobra.Command {
+	return &cobra.Command{
+		Use:   addCopy,
+		Short: short,
+		Long:  desc,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return addAndCopyCmd(cmd, args, strings.ToUpper(addCopy), *opts)
```

Comment:
Doesn't this still need a boolean true before *opts ala:
```suggestion
			return addAndCopyCmd(cmd, args, strings.ToUpper(addCopy), true, *opts)
```

Reply:
Oh wait, I see you did away with the boolean, but it looks like you're always doing an ADD and I think you still need to differentiate and do an ADD or COPY.

Reply:
It is adressed later in the code. Please see the line 124:
extractLocalArchives := verb == "ADD"

Reply:
Oh, that's subtle.  I didn't catch the createCommand() call setting verb to either ADD or COPY above.